config/jobs/kubernetes/sig-arch/conformance-gate.yaml

```diff
@@ -0,0 +1,18 @@
+periodics:
+- interval: 4h
+  name: conformance-gate
+  annotations:
+    testgrid-dashboards: conformance-apisnoop
+    testgrid-tab-name: conformance-gate
+    test-grid-alert-email: en-server-prow-test-alert@google.com
```

Comment:
This should be the sig arch mailing list?

Reply:
There should be a description annotation explaining with a couple of sentences what the job does. Example https://github.com/kubernetes/test-infra/blob/b660a5251071db1e5aca5a9e1ca5b08064691eb7/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L5

Reply:
Agree with @neolit123's comments, although I might hold off on alerting the arch mailing list until this is proven to run and not fail spuriously.  The existing e-mail address is not valid though, I'd maybe suggest apisnoop@ii.coop to start with?

If the goal is to say sig arch owns this job, it would be helpful to setup a sig-arch dashboard group, eg: sig-arch/sig-arch-conformance/apisnoop-conformance-gate or something similar.

Reply:
@spiffxp and @neolit123 : I've included the creation of a sig-arch dashboard into this pr and updated the job's annotations accordingly.  The dashboard group is `sig-arch`, and the dashboard-name is `sig-arch-conformance`.